GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -396,6 +400,11 @@ private void UpdateSelectedFileViewers(bool force = false)
                 }
             }
 
+            if (tabControl1.SelectedTab?.Parent is null)
+            {
+                tabControl1.SelectedTab = DiffTab;
+            }
```

Comment:
What is the purpose of the `DiffTab` in FormFileHistory?
1. It just duplicates what can be seen on the `CommitTab` - with a little more space, of course.
2. It does not really apply when showing the history of a folder.

So, it should be hidden as the `BlameTab` and the `ViewTab` - or be removed completely.

Reply:
Diff tab should show all changes to the folder. GE parsing fails to present this, probably the only place that is used.
To fix the parsing would require that the path is specially handled, which could be done, but as the info is available I will remove that.